addons/config/configs/CfgWeapons.hpp

```diff
@@ -232,10 +232,11 @@ class CfgWeapons
                     passThrough = 0.2; // 0.3
                 };
             };
-            mass = 286.52; // 100
+            containerClass = "Supply160"; // "Supply120"
+            mass = 308.56; // 80
         };
-        descriptionShort = "Panzerungsstufe IV"; // "Sprengstoffresistent"
-        displayName = "SPS Tier 4 (UCP)"; // "Boden-Luft-Granatwerfer-Tragegurt (Tarnfarbe)"
+        descriptionShort = "Panzerungsstufe V"; // "Sprengstoffresistent"
+        displayName = "SPS (UCP)"; // "Boden-Luft-Granatwerfer-Tragegurt (Tarnfarbe)"
```

Comment:
Bin der Meinung das nur der `displayName` im Arsenal angezeigt wird und die `descriptionShort` nur im Inventar, das heißt man wüsste nicht welche Stufe die sind. Ggf in die ArsenalStatistic?!?

Reply:
Vollkommen richtig, habe nicht dran gedacht, dies auch bei der SPS Weste zu machen, weil die bisher stets die schwerste war. Wird angepasst.